src/main/java/com/revealprecision/revealserver/api/v1/facade/service/TaskFacadeService.java

```diff
@@ -29,13 +50,30 @@
 
   private final TaskFacadeProperties taskFacadeProperties;
 
+  private final TaskService taskService;
+
+  private final ActionService actionService;
+
+  private final PlanService planService;
+
+  private final PersonService personService;
+
+  private final LocationService locationService;
+
   @Autowired
   public TaskFacadeService(MetaDataJdbcService metaDataJdbcService, UserService userService,
```

Comment:
just a suggestion, if we introduce @RequiredArgsConstructor to class, we can get rid of this boilerplate code.(nothing wrong with this, but will result in less code)

Reply:
thanks